### DISCUSSION TITLE
如何计算两个多项式的除法

### DISCUSSION THREAD

**Discussion by sgz13140** [coset-io/zkp-academy#11] [Q&A]
比如应该如何计算f(x)/g(x)

**Comment by 0xxEric** [comment]
1、如果是指工程上如何实践的话，应该是在代码层面，有对两个多项式进行除法的计算（这个请有经验的大佬可以补充详细细节），或者跟一下，看代码的实现细节。 例如，galois库中，就有实现多项式相除的运算。

2、如果是指数学层面，为何两个能相除的话。这里应该是在Galois域上的运算。这个文章中以GF(2^n)为例，举例说明。
https://www.cnblogs.com/xdyixia/p/12576437.html
当然，代码实现应该也是基于Galois域上的运算的原理。这个我还没有详细研究。
